vmdb/app/controllers/ops_controller/settings/cap_and_u.rb

```diff
@@ -392,7 +392,7 @@ def build_ds_tree(storages)
       ds_node[:key] = "Datastore_" + s[:id].to_s
       name = "<b>#{s[:name]}</b> [#{s[:location]}]"
```

Comment:
Does this local variable `name` still make sense or should it be collapsed into the RHS of `ds_node[:title]`?